lib/scripts/pdf_a4_portrait.js

```diff
@@ -153,11 +154,19 @@ function createSection (section, content, options) {
     height: options.height,
     contents: phantom.callback(function (pageNum, numPages) {
       var html = o[pageNum] || c[pageNum]
+
+      var pageNumFinal = pageNum, numPagesFinal = numPages;
+      if(opts.paginationStartPage && opts.paginationStartPage > 1) {
+        var pageOffset = opts.paginationStartPage - 1;
+        pageNumFinal = (pageNumFinal - pageOffset > 0 ? pageNumFinal - pageOffset : '');
+        numPagesFinal -= pageOffset;
```

Comment:
Agreed, will make those changes.
Also would be nice if we could have header and footer for the current page. E.g. if you have, in a .page element a .header or .footet, it be used as the header/footer for that specific page.